src/GitHook/Command/FileCommand/PreCommit/TsLintCommand.php

```diff
@@ -0,0 +1,107 @@
+<?php
+
+/**
+ * MIT License
+ * For full license information, please view the LICENSE file that was distributed with this source code.
+ */
+
+namespace GitHook\Command\FileCommand\PreCommit;
+
+use GitHook\Command\CommandConfigurationInterface;
+use GitHook\Command\CommandInterface;
+use GitHook\Command\CommandResult;
+use GitHook\Command\CommandResultInterface;
+use GitHook\Command\Context\CommandContextInterface;
+use GitHook\Helper\ProcessBuilderHelper;
+
+class TsLintCommand implements CommandInterface
+{
+    use ProcessBuilderHelper;
+
+    protected const MERCHANT_PORTAL_MODULES = ['ZedUi', 'MerchantPortalGui'];
```

Comment:
is it a way to do this a little bit more extendable? How we prepared current list?

Reply:
Please rename constant to `MERCHANT_PORTAL_PATTERNS`